packages/mui-joy/src/Select/Select.tsx

```diff
@@ -241,7 +241,9 @@ const SelectListbox = styled(StyledList, {
       theme.vars.palette.background.popup,
     '--List-item-stickyTop': 'calc(var(--List-padding, var(--List-divider-gap)) * -1)', // negative amount of the List's padding block
     ...scopedVariables,
-    minWidth: 'max-content', // prevent options from shrinking if some of them is wider than the Select's root.
+    maxHeight: 300, 
+    overflow: 'auto',
+    width: 'max-content', // prevent options from shrinking if some of them is wider than the Select's root.
```

Comment:
@Vivek-Prajapatii May I ask why `minWidth` is changed to `width`?

Reply:
<img width="329" alt="image" src="https://user-images.githubusercontent.com/18292247/220277666-d91be22f-43bc-4a88-9fd5-91587443d4ab.png">

`width` will be overridden by Popperjs, so it should be `minWidth`.

Reply:
Thank you @siriwatknp for pointing this out, happened by mistake.